man/systemd.netdev.xml

```diff
@@ -459,7 +459,7 @@
         <listitem>
           <para>Defines a mapping of Linux internal packet priority (<constant>SO_PRIORITY</constant>) to VLAN header
           PCP field for outgoing frames. Takes a whitespace-separated list of unsigned integer pairs in the format
-          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1–4294967294.
+          <literal>from</literal>-<literal>to</literal>, e.g., <literal>21-7 45-5</literal> ranges 1…4294967294.
```

Comment:
I don't understand this sentence at all. Is "ranges" used in the sense of "is equivalent to" ?